controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java

```diff
@@ -286,6 +287,8 @@ private HttpResponse handleGET(Path path, HttpRequest request) {
 
     private HttpResponse handlePUT(Path path, HttpRequest request) {
         if (path.matches("/application/v4/tenant/{tenant}")) return updateTenant(path.get("tenant"), request);
+        if (path.matches("/application/v4/tenant/{tenant}/access/ssh/request")) return requestSshAccess(path.get("tenant"), request);
+        if (path.matches("/application/v4/tenant/{tenant}/access/ssh/approve")) return approveAccessRequest(path.get("tenant"), request);
```

Comment:
Need to limit this API to tenant (administrators(?)) only in `controller-api` (PathGroup|Policy|RoleDefinition)

Maybe consider ordering path arguments so that approve is before ssh, then we can limit access to all ../access/* APIs